README.md

```diff
@@ -160,6 +160,10 @@ api.post('/users', options)
 // }
 ```
 
+## Logging
+
+The library uses [debug](https://github.com/visionmedia/debug) for logging. Install the `debug` library in your project and add `snuffles:requests`, `snuffles:responses`, or `snuffles:*` (for both) to your debug namespaces config (e.g. `DEBUG=snuffles:*`) in order to see Snuffles log entries appear in your logs. If the logs should get stored anywhere, be careful that you could be exposing sensitive information like passwords, API tokens, etc.
```

Comment:
Do we need to implement it specifically for `debug`?

With the current implementation, you could just use `{ logger: console.log }` - which I find quite useful.

Or how about something like


```
{
  logger: (type, ...data) => { ... }
}
```

Then one could use `debug` with:

```
{
  logger: (type, ...data) => debug(`snuffles:${type}`)(...data)
}
```

and you still could use it with console.log

```
{ logger: console.log }
```

Reply:
Ah, okay, sorry, GitHub didn't refresh the page...

Reply:
find ich gut, hab ich so übernommen